python/tests/yarn.lock

```diff
@@ -0,0 +1,67 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
You might have run `yarn add` in this folder which is why it made this `yarn.lock` in the `python/tests` folder (guessing it shouldn't be here?)

Reply:
That was a remnant of my tests, removed